guides/builder.md

```diff
@@ -5,20 +5,23 @@ This is a feature that makes possible to build numscripts dynamically within you
 You just need to call `Numscriptex.Builder.build/1` with the metadata necessary to build your numscript.
 
 ### Metadata
-Metadata is the sole argument you need to use the `build/1` function, wich have the following fields:
+Metadata is the sole argument you need to use the `build/1` function, wich has the following fields:
 
 Required fields:
 - splits: a list of metadata needed to build numscripts (explained below)
 
 Optional fields:
 - percent_asset: monetary asset (BRL, USD, EUR, and etc.) to use when build numscripts using percentage values.
-- remaining_to¹: where the rest of the money will go after the transaction
-  - Only needed if you want to designate an specific destination, the default is `remaining kept` wich makes the remainder go back to the source.
+- remaining_to: where the rest of the money will go after the transaction is made
+  - Only needed if you want to designate a specific destination. The default is [remaining kept](https://docs.formance.com/numscript/reference/destinations#allocation-destinations) wich makes the remainder go back to the source.
 
 About the "list of metadata" mentioned on the `split` field above:
+
 Required fields:
 - account: account whose the money will go to
-- amount: amount of money transferred (accepts both integer and float)
+- amount: amount of money transferred (accepts both integer and float), but be aware that:
+  - this field is treated as the literal amount you want to send (e.g. if yout want to send $5, your amount field must be 5 or 5.0)
+  - Numscript treats its amounts as integers insted of literals like `Numscriptex`(e.g. $5 would be 500), so the script generated will send 500 instead of 5.
```

Comment:
Depois de explicar isso aqui não sei se é lá uma boa UX, já que valor monetario no input é tratado de um jeito e no output de outro totalmente diferente. Acha melhor deixar como integer ($5 como 500) ou como ta mesmo?

Reply:
Se for para ter sempre igual, acho melhor ter tudo como integer e esquecer dos decimais. É mais fácil usar decimal, mas complicaria muito.